cg/cli/set/families.py

```diff
@@ -0,0 +1,77 @@
+import click
+from .family import family
+from cg.constants import FAMILY_ACTIONS, PRIORITY_OPTIONS
+from cg.store import models, Store
+
+CONFIRM = "Continue?"
+
+
+def _get_samples_by_identifiers(
```

Comment:
Love type hints when they are this explicit 😸

Reply:
👍